### PR TITLE
Add Telegram Bot

### PR DESCRIPTION
**Pull request by theapache64**
+ Fix README typos.

### CODE REVIEW

File: covid-frontend/.env [hunk 1]
```diff
@@ -1,2 +0,0 @@
-REACT_APP_ENV=development
```
Comment (tholor) [inline covid-frontend/.env:1]: @sfakir can we get rid of these here or what is are they used for?

Comment (theapache64) [reply covid-frontend/.env:1]: @tholor  The `.env` file is ignored in the [.gitignore](https://github.com/deepset-ai/COVID-QA/blob/b0588f38b051b7ab89cedb3ff184878f8674538d/covid-frontend/.gitignore#L11) but wasn't removed from tracking. That's why I removed it.

Comment (sfakir) [reply covid-frontend/.env:1]: yes, the .env file can be removed, I think it's obvious but I will add to the readme that a dev has to copy a sample .env to connect to the backend (local or staging)

**Comment by tholor** [comment]
Thanks again for creating this PR. Great work!

Two comments: 
- Right now the answer displayed by the bot doesn't contain the "matched question". However, it might be helpful for the users to see that in order to judge if the answer is really relevant to their question. You find it in the response JSON in the field "question". 
- We now also have the option for user feedback (see [API endpoint](https://github.com/deepset-ai/COVID-QA/blob/master/backend/controller/feedback.py)). So people can rate if the given answer was helpful or not and we will use the data to improve the NLP model. This could be also an helpful addition to the telegram bot. 

Would be great to hear your thoughts on that and maybe address them in a separate PR.

**Review by tholor — APPROVED** [review]

**Comment by theapache64** [comment]
> Right now the answer displayed by the bot doesn't contain the "matched question". However, it might be helpful for the users to see that in order to...

Okay. That'll be good. I can include the matched question.

> We now also have the option for user feedback (see API endpoint). So people can rate if the given answer was helpful or ...

Definitely. I'll include that as well.

I'll start working on these requirements and will let you know once it's done.

**Pull request merged**